test/Integration/test-js/AddProductIncorrectCasing/index.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
copyright (could you follow up with adding linting for the test files too? Looks like it's just doing the samples folder currently)

Reply:
Will send a follow up PR for linting test-js folder